bucket/p4v.json

```
@@ -0,0 +1,44 @@
+{
+    "version": "2021.4/2227050",
+    "description": "Helix Visual Client (P4V) is a desktop app that provides access to versioned files in Helix Core through a graphical interface.",
+    "homepage": "https://www.perforce.com/products/helix-core-apps/helix-visual-client-p4v",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://www.perforce.com/perforce/doc.current/user/p4v_license.txt"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://cdist2.perforce.com/perforce/r21.4/bin.ntx64/p4vinst64.msi",
+            "hash": "6f423b7e58a92faa1b2a463c33209cb1c86a638dad2f1cddabd851d436e2cd96"
+        }
+    },
+    "extract_dir": "Perforce",
+    "shortcuts": [
+        [
+            "p4v.exe",
+            "P4V"
+        ],
+        [
+            "p4merge.exe",
+            "P4Merge"
+        ],
+        [
+            "p4admin.exe",
+            "P4Admin"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.perforce.com/support/software-release-index",
+        "regex": "Helix Visual Client, Helix Merge, and Helix Admin.*?(20(?<major>\\d+)\\.(?<minor>\\d+)(?:\\.\\d+)*\\/\\d+)"
```

```suggestion
        "regex": "Helix Visual Client, Helix Merge, and Helix Admin.*?20(?<major>\\d+)\\.(?<minor>\\d+)(?:\\.\\d+)*\\/(?<build>\\d+)",
        "replace": "20${major}.${minor}-${build}"
```
*Scoop* does not support `/` in version numbers, therefore we need a **replace** to make it work.